### PR TITLE
Changed infraction length

### PR DESCRIPTION
**Pull request by Arlodotexe**

### CODE REVIEW

File: src/bot/commands/infraction.ts [hunk 1]
```diff
@@ -250,31 +250,31 @@ async function initExistingInfractionData(server: Guild) {
...
             role: warnedRole,
-            expiresAfterDays: 3
+            expiresAfterDays: 14
         },
         {
             label: "Strike 1",
             role: strike1Role,
-            expiresAfterDays: 7,
-            unmuteAfterDays: 1
+            expiresAfterDays: 21, // 2 weeks
```
Comment (yair100) [inline src/bot/commands/infraction.ts]:
```suggestion
            expiresAfterDays: 21, // 3 weeks
```

Comment (Arlodotexe) [reply src/bot/commands/infraction.ts]: Good catch!

**Review by sylveon — APPROVED** [review]

File: src/bot/commands/infraction.ts [hunk 2]
```diff
@@ -250,31 +250,31 @@ async function initExistingInfractionData(server: Guild) {
...
-            unmuteAfterDays: 1
+            expiresAfterDays: 21, // 3 weeks
+            unmuteAfterDays: 7
         },
         {
             label: "Strike 2",
             role: strike2Role,
-            expiresAfterDays: 14,
-            unmuteAfterDays: 3
+            expiresAfterDays: 63, // ~2 months
```
Comment (yaichenbaum) [inline src/bot/commands/infraction.ts:264]:
```suggestion
            expiresAfterDays: 63, // ~3 months
```

Comment (Sergio0694) [reply src/bot/commands/infraction.ts:264]: Uh... Aren't 63 days about 2 months though?

**Review by Sergio0694 — APPROVED** [review]
Looks good 👍

**Review by yair100 — APPROVED** [review]

**Pull request merged**